spec/yaml/swiftnav/sbp/bootload.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Fergus Noble <fergus@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.bootload
+description: |
+  Messages for the bootloading configuration on the Piksi. These are
+  in the implementation-defined range (0x0000-0x00FF), and intended
+  for internal-use only. Note that some of these messages taking a
+  request from a host and a response from the Piksi share the same
+  message type ID.
+stable: False
+public: True
+include:
+  - types.yaml
+definitions:
+
+ - MSG_BOOTLOADER_HANDSHAKE:
+    id: 0x00B0
+    public: True
+    short_desc: Bootloading handshake (Host <=> Piksi).
+    desc: |
+      The host issues a handshake message prior to flashing Piksi's
+      firmware.
+    fields:
+      - handshake:
+          type: u8
+          desc: Handshake value
```

Comment:
On old versions of the bootloader (<=v0.1), hardcoded u8=0. On new versions, return the git describe string for the bootloader build.